### PR TITLE
IOS-713: Adjustments for smaller screens

### PR DESCRIPTION
**Pull request by Malibris**
### Ticket
[IOS-713](https://netguru.atlassian.net/browse/IOS-713)

### Task Description
AS IS
https://monosnap.com/file/x4cqTvY67nnPBV2Aylr3o3oTOiBblT
if I choose all stream sources (4), only two streams are visible on the main page.
TO BE
I should see all stream sources.

### Aditional Notes (optional)
As a temporary solution, instead of all stream sources displayed in case of choosing 'my set' option, a simple 'my set' label is displayed.

### CODE REVIEW

File: Inbbbox/Source Files/Views/Custom/ShotsCollectionBackgroundView.swift [hunk 1]
```diff
@@ -5,6 +5,11 @@
 import PureLayout
 import UIKit
 
+struct ScreenSize {
+    static let iPhone4OrLess = UIScreen.main.bounds.height < 568
+    static let iPhoneSE = UIScreen.main.bounds.height == 568
+}
+
```
Comment (kkapitan) [inline Inbbbox/Source Files/Views/Custom/ShotsCollectionBackgroundView.swift]: Nie da się tego zrobić używając DeviceKit?

Comment (Malibris) [inline Inbbbox/Source Files/Views/Custom/ShotsCollectionBackgroundView.swift]: It's a workaround for detecting the app running in an iPhone simulated environment on iPad. So, the name of the first option is a little misleading, but also it is only a temporary solution. I'll remove it when iPad version will be finished.

Comment (kkapitan) [inline Inbbbox/Source Files/Views/Custom/ShotsCollectionBackgroundView.swift]: (also I think that PR title should be changed)

Comment (Malibris) [inline Inbbbox/Source Files/Views/Custom/ShotsCollectionBackgroundView.swift]: Yup, that's my mistake.

**Pull request merged**